server/views/components/search/index.njk

```diff
@@ -0,0 +1,33 @@
+{% if featuresCohort | isFlagEnabled('searchWorks', featureFlags) %}
+  {% set action = '/search' %}
+  {% set method = 'get' %}
+  {% set name = 'query' %}
+{% else %}
+  {% set action = 'https://wellcomecollection.org/search' %}
+  {% set method = 'post' %}
+  {% set name = 'search' %}
+{% endif %}
+
+<div id="header-search" class="header__search">
+  <form class="header__form"
+    action="{{action}}"
+    method="{{method}}">
+    <div class="header__input-wrap js-show-hide-drawer">
+      <label class="header__label" for="header-input">Search</label>
+      <input id="header-input" class="header__input js-header-input" placeholder="Search" name="{{name}}" />
+      {% if not featuresCohort | isFlagEnabled('searchWorks', featureFlags) %}
+      {# This is here as we're sending people back to V1, and drupal requires it #}
+      {# TODO: remove once we build our own search interface #}
+      <input type="hidden" name="form_id" value="col_search_form" />
+      {% endif %}
+    </div>
+    <button class="header__button js-show-hide-trigger">
+      <span class="header__button-inner">
+        {% icon 'actions/search', null, ['header__search-button'] %}
+        <span class="header__button-text">Search</span>
+    </button>
+    <span role="button" aria-controls="header-input" class="header__close-search js-header-close-search">
+      {% icon 'actions/cross-big', 'Close search box' %}
+    </span>
+  </form>
+</div>
```

Comment:
If we're worried about breaking things in prod, we could just have two components, `search` and `image-search` - and just copy the component over and include the relevant one from the parent.

Makes it way less brittle.